css/revops.css

```diff
@@ -259,6 +259,10 @@ p {
   display: grid;
 }
 
+.blog-button {
```

Comment:
This could probably be made more generic, so it can be reused. Describe what it does instead of where it is used (that's actually a good rule of thumb for components in CSS, whereas page-specific styling can be named for where it appears)